fuzz/helper.py

```diff
@@ -9,13 +9,16 @@
 
 """Fuzzing helper, creates and uses corpus/crash directories.
 
-fuzzer.py <fuzzer> <extra fuzzer arguments>
+helper.py <fuzzer> <extra fuzzer arguments>
 """
 
 import os
 import subprocess
 import sys
 
+if len(sys.argv) < 2:
+    exit("Please provide a <fuzzer> argument as one of the executables in fuzz/")
+
```

Comment:
In this addition a safety check was added to let the user know why the index is out of bounds when trying to access the arguments in `sys.argv[1]`. 
```python
FUZZER = sys.argv[1]

THIS_DIR = os.path.abspath(os.path.dirname(__file__))
CORPORA_DIR = os.path.abspath(os.path.join(THIS_DIR, "corpora"))

FUZZER_DIR = os.path.abspath(os.path.join(CORPORA_DIR, FUZZER))
if not os.path.isdir(FUZZER_DIR):
    os.mkdir(FUZZER_DIR)
```